cartography/intel/aws/securityhub.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
```

Comment:
Also decorate with `@aws_handle_regions` from `cartography.util`

Reply:
This isn't regional. It's per account.